src/components/MenuItem.js

```diff
@@ -122,6 +122,8 @@ function MenuItem(props) {
             ]}
             disabled={props.disabled}
             ref={props.forwardedRef}
+            accessibilityRole={CONST.ACCESSIBILITY_ROLE.MENUITEM}
+            accessibilityLabel={props.title}
```

Comment:
props.title was of number `0` in some cases which caused app crash as mentioned here https://github.com/Expensify/App/issues/25451#issuecomment-1683356992.
We should make sure the values we pass are string.